Cargo.toml

```diff
@@ -31,23 +31,23 @@ env_logger = "0.9.0"
 hex_fmt = "0.3.0"
 init_with = "1.1.0"
 log = "0.4.17"
-rand = "0.7"
+rand = "0.8.5"
 rand_derive = "0.5.0"
 reed-solomon-erasure = "5.0.3"
 serde = { version = "1.0.137", features = ["derive", "rc"] }
 thiserror = "1.0.31"
-threshold_crypto = { git = "https://github.com/fedimint/threshold_crypto" }
+threshold_crypto = { git = "https://github.com/elsirion/threshold_crypto", branch = "2022-08-upgrade-pairing" }
```

Comment:
We probably want to change this back now after fedimint/threshold_crypto#6 got merged?